### PR TITLE
Add Export/Import functionality in Permission UI

### PR DESCRIPTION
**Pull request by cbadura**
- Add Export and Import actions (as buttons) on app-search components
- Labels/Tooltips/Base functionality
- Add actions: Export, Import
- Add export dialog where the user can select the application to be exported
with ALL (means all applications)
- Add import dialog where the user can select a file for uploading
- Bump deps

### CODE REVIEW

File: src/app/permission/app-search/app-search.component.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
-<ocx-portal-page helpArticleId="PAGE_PERMISSION_SEARCH">
+<ocx-portal-page permissionArticleId="PAGE_PERMISSION_SEARCH">
```
Comment (HenryT-CG) [inline src/app/permission/app-search/app-search.component.html:1]: mmmh, I'm quite sure that change is not correct

File: src/app/permission/app-search/app-search.component.spec.ts [hunk 2]
```diff
@@ -14,10 +13,14 @@ import {
...
   WorkspacePageResult,
   ApplicationPageResult,
-  Application
+  Application,
+  AssignmentAPIService,
+  Permission
 } from 'src/app/shared/generated'
 import { App, AppSearchComponent } from './app-search.component'
 import { RowListGridData } from '@onecx/angular-accelerator'
+// import { FileSelectEvent } from 'primeng/fileupload'
```
Comment (HenryT-CG) [inline src/app/permission/app-search/app-search.component.spec.ts:22]: do we need this import?

File: src/app/remotes/user-roles-permissions/user-roles-permissions.component.spec.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing'
-import { NO_ERRORS_SCHEMA } from '@angular/core'
+import { ElementRef, NO_ERRORS_SCHEMA } from '@angular/core'
```
Comment (HenryT-CG) [inline src/app/remotes/user-roles-permissions/user-roles-permissions.component.spec.ts:2]: imports from @angular/core should be the first import in the file

Comment (cbadura) [reply src/app/remotes/user-roles-permissions/user-roles-permissions.component.spec.ts:2]: done

File: src/app/permission/app-search/app-search.component.spec.ts (hunk 2 above)
Comment (cbadura) [reply src/app/permission/app-search/app-search.component.spec.ts:22]: I wasn't able to get the related test to run and commented it out. That's why this is also commented out

File: src/app/permission/app-search/app-search.component.html (hunk 1 above)
Comment (cbadura) [reply src/app/permission/app-search/app-search.component.html:1]: It should be this: <ocx-portal-page permission="PAGE_PERMISSION_SEARCH"> ?

**Review by HenryT-CG — APPROVED** [review]

**Pull request merged**